fetchmail_attach_from_folder/wizard/attach_mail_manually.py

```diff
@@ -96,12 +96,12 @@ class AttachMailManuallyMail(models.TransientModel):
     _name = "fetchmail.attach.mail.manually.mail"
     _description = __doc__
 
-    wizard_id = fields.Many2one("fetchmail.attach.mail.manually", readonly=True)
-    msgid = fields.Char("Message id", readonly=True)
-    subject = fields.Char(readonly=True)
-    date = fields.Datetime(readonly=True)
-    email_from = fields.Char("From", readonly=True)
-    body = fields.Html(readonly=True)
+    wizard_id = fields.Many2one("fetchmail.attach.mail.manually")
```

Comment:
Why are you removing the `readonly=True` here and elsewhere?

Reply:
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0

> If a field is marked as readonly=True on the model, it won't be possible to import it through the Odoo import tool, so avoid it as possible and define the readonly attribute on the views instead.

But I just realized that I didn't put them in the view, when I can I fix it.